modules/store/spec/types/utils.ts

```diff
@@ -2,6 +2,7 @@ export const compilerOptions = () => ({
   moduleResolution: 'node',
   target: 'es2015',
   baseUrl: '.',
+  experimentalDecorators: true,
```

Comment:
Added for debug purposes, because `expectSnippet().toSucceed()` gives the error on the `@Injectable` decorators.

Reply:
this comment should be in the file itself :)

Reply:
🤯 I usually just putted the comment in the PR, but it now that I read this comment it's indeed better to add it in the file. Thanks for the tip!